### ISSUE TITLE
Question: how does this compare to devbox?

### ISSUE THREAD

**Issue by vsoch** [cachix/devenv#43]
Specifically, this is devbox: https://github.com/jetpack-io/devbox

I'm loving nix (and have started using devbox) and am wondering how the two compare. Thanks!

**Comment by domenkozar** [comment]
Hey! I'm working on a FAQ and will post it on the website soon.

Quickly, the main differences are:

- devbox uses JSON for specification, while devenv uses Nix. Using a real language brings a lot of power that is delegated to code generation in devbox. Since we have https://nix.dev/tutorials/nix-language I think this is totally worth it.
- devbox focuses on containers, while devenv wants to empower local development. 
- devbox uses code generation, which is extremely suboptimal way to scale this.
- We have thousands of Nix developers that are already using different ways to create local dev environments and embracing the power of the community will lead us to supporting pretty much anything.
- devenv composes **extremely** well, for example [0.2](https://devenv.sh/blog/2022/11/14/devenv-02/) added support for ``devenv.local.nix``.

TLDR: using Nix as a powerful language that made all this possible is far more superior to a JSON format.

As time passes this will become more obvious :)

**Comment by vsoch** [comment]
Thank you - this is perfect! Going to give it a try now <3
